Px.Utils/Validation/SyntaxValidation/Bcp47Codes.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Immutable;
+
+namespace PxUtils.Validation.SyntaxValidation
+{
+    /// <summary>
+    /// Provides a list of BCP-47 language codes. BCP-47 is a standard for the identification of human languages.
+    /// </summary>
+    public static class Bcp47Codes
+    {
+        /// <summary>
+        /// An immutable hash set of BCP-47 language codes.
```

Comment:
No need to restate the type in documentation. It is already visible and can only be outdated if the type changes. "Set of abc.." is enough.